docs/auth/federated-auth.md

```diff
@@ -24,7 +24,7 @@ Ensure the "Google" sign-in provider is enabled on the [Firebase Console](https:
 > change to Google, due to Firebase Authentications concept of trusted providers. You can find out more about
 > this [here](https://groups.google.com/g/firebase-talk/c/ms_NVQem_Cw/m/8g7BFk1IAAAJ).
 
-* {iOS+ and Android}
+- {iOS+ and Android}
```

Comment:
I'm not sure on changes to style 🤔. This documentation is rendered on [firebase docs here](https://firebase.google.com/docs/auth/flutter/federated-auth) and I don't know how this might affect their layout.

Reply:
Thanks, I've changed it back